lib/org/chromium/webidl/ast/Argument.js

```diff
@@ -24,6 +24,12 @@ foam.CLASS({
       class: 'Boolean',
       name: 'isVariadic',
     },
+    {
+      class: 'String',
+      documentation: 'A unique identifier used by the Diff algorithm.',
```

Comment:
This is good. Let's mention *why* ('cause, diff algorithm) in all cases where we're adding ids.